ir/memory.cpp

```diff
@@ -348,4 +363,12 @@ Memory Memory::mkIf(const expr &cond, const Memory &then, const Memory &els) {
   return ret;
 }
 
+expr Memory::isBlockAlive(const expr &bid) {
```

Comment:
what do you think about this and the next function to the pointer class instead?

Reply:
I have no special preference, I'll move them to Pointer.